nekoyume/Assets/_Scripts/UI/Scroller/GrindingItemSlotCell.cs

```diff
@@ -0,0 +1,50 @@
+using Nekoyume.UI.Model;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Nekoyume.UI.Scroller
+{
+    public class GrindingItemSlotCell : GridCell<InventoryItem, GrindingItemSlotScroll.ContextModel>
+    {
+        [SerializeField]
+        private Image itemImage;
+
+        [SerializeField]
+        private Animator animator;
+
+        [SerializeField]
+        private GameObject selectedObject;
+
+        [SerializeField]
+        private GameObject noneObject;
+
+        [SerializeField]
+        private Button removeButton;
+
+        private InventoryItem item;
```

Comment:
```suggestion
        private InventoryItem _selectedItem;
```
I think it would be better if it was a more accurate name. And please put '_' on the private field.